catalyst_voices/lib/pages/registration/account_completed/account_completed_panel.dart

```diff
@@ -0,0 +1,270 @@
+import 'package:catalyst_voices/common/ext/account_role_ext.dart';
+import 'package:catalyst_voices/pages/registration/widgets/next_step.dart';
+import 'package:catalyst_voices/routes/routing/account_route.dart';
+import 'package:catalyst_voices/routes/routing/spaces_route.dart';
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class AccountCompletedPanel extends StatelessWidget {
+  const AccountCompletedPanel({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Expanded(
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                const _TitleText(),
+                const SizedBox(height: 10),
+                BlocBuilder<RegistrationCubit, RegistrationState>(
+                  builder: (context, state) {
+                    final roles =
+                        state.walletLinkStateData.selectedRoles?.toList() ?? [];
+                    final walletName =
+                        state.walletLinkStateData.selectedWallet?.wallet.name ??
+                            '';
```

Comment:
Looks like `BlocWalletLinkBuilder`

Reply:
I've used BlocSelector as Dominik suggested. Lmk if that works for you too.